<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DownloadDialog</class>
 <widget class="QDialog" name="DownloadDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1082</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Download files</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="image_lbl">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="resource.qrc">:/images/mplogo.png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="intro_lbl">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Below are files available for download from Memory Portraits. Select the files you want to download and press Download.</string>
         </property>
         <property name="textFormat">
          <enum>Qt::RichText</enum>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::NoTextInteraction</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="table_widget">
       <property name="columnCount">
        <number>7</number>
       </property>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Name</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Order ID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>File Size</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>File ID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Uploaded</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Last download</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Expires</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="debug_text_ed"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="help_btn">
         <property name="text">
          <string>&amp;Help</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="messages_btn">
         <property name="text">
          <string>&amp;Messages</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="close_btn">
         <property name="text">
          <string>&amp;Close</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="download_btn">
         <property name="text">
          <string>&amp;Download</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>table_widget</tabstop>
  <tabstop>debug_text_ed</tabstop>
  <tabstop>help_btn</tabstop>
  <tabstop>messages_btn</tabstop>
  <tabstop>close_btn</tabstop>
  <tabstop>download_btn</tabstop>
 </tabstops>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>close_btn</sender>
   <signal>clicked()</signal>
   <receiver>DownloadDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1028</x>
     <y>599</y>
    </hint>
    <hint type="destinationlabel">
     <x>540</x>
     <y>313</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
